src/app/settings.py

```diff
@@ -153,6 +161,15 @@
 
     sentry_sdk.init(dsn=env("SENTRY_DSN"), integrations=[DjangoIntegration()])
 
+
+if ENVIRONMENT == "development":
+
+    def show_toolbar(request):
+        return True
+
+    DEBUG_TOOLBAR_CONFIG = {"SHOW_TOOLBAR_CALLBACK": show_toolbar}
```

Comment:
We can get benefit from lambda functions instead of creating a function just to return a `True` statement. What do u think?

```suggestion
    DEBUG_TOOLBAR_CONFIG = {"SHOW_TOOLBAR_CALLBACK": lamda: True}
```

Reply:
@rodolfolottin We would need a `request` parameter in order to make it work:
```suggestion
    DEBUG_TOOLBAR_CONFIG = {"SHOW_TOOLBAR_CALLBACK": lambda request: True}
```

Reply:
Ops, I forgot the argument! Thanks @fredericojordan

Reply:
NIce